main/ertp/guide/create-specific-erights.md

```diff
@@ -0,0 +1,87 @@
+# Creating a specific eright with ERTP
+
+## Definition of "specific"
+
+In the context of ERTP and assets, "specific" means "non-fungible".
+
+Assets of the same assay are not fungible if each different item of the assay
+cannot be replaced with another.
+House ownership rights are not fungible, because each ownership right refers to a 
+specific house and one house is not equal to another.
+
+## Creating a fungible asset with ERTP
+
+### Creating and sending the asset
+
+In ERTP, digital assets are created by a [`mint`](./mint.html). Having access to the mint
+gives you the power to create more digital assets of the same type at
+will.
+
+Let's say we own an Opera and want to sell tickets to seats for ballet shows. Tickets are
+the specific assets we want to represent. Tickets refer to a specific seat for a specific 
+show at a specific time and date.
+
+To do that, you would first install the [ertp JavaScript package](https://www.npmjs.com/package/@agoric/ertp)
+(`npm install @agoric/ertp`) and then:
+
+```js
+import { makeMint } from '@agoric/ertp/core/mint';
+
+import harden from '@agoric/harden';
+
+import { noCustomization } from '@agoric/ertp/core/config/noCustomization.js';
+import { makeCoreMintKeeper } from '@agoric/ertp/core/config/coreMintKeeper';
+
+import { insist } from '@agoric/ertp/util/insist';
+import { mustBeComparable } from '@agoric/ertp/util/sameStructure';
+
+const insistOptDescription = optDescription => {
+  insist(!!optDescription)`optDescription must be truthy ${optDescription}`;
+  mustBeComparable(optDescription);
+};
+
+function makeInviteConfig() {
+  return harden({
+    ...noCustomization,
+    makeMintKeeper: makeCoreMintKeeper,
+    extentOpsName: 'uniExtentOps',
```

Comment:
I would have thought this would use the ListExtentOps, so that you can hold a purse with multiple tickets in it. Was there a reason not to? I think the ListExtentOps makes more intuitive sense to beginners, whereas the uniExtents aren't meant to be combined, so they're a bit strange.

Reply:
I copied the code from the invite mint from contractHost and didn't question it enough
I agree with you, ListExtentOps makes more sense

Reply:
that said, `listExtentOps` is not exposed in 
https://github.com/Agoric/agoric-sdk/blob/e4b7c868858329928c7fb25f4cac881d81458a99/packages/ERTP/core/config/extentOpsLib.js

And in the current ERTP code, a UnitOps can only ever be created from things exposed in extentOpsLib.js

https://github.com/Agoric/agoric-sdk/blob/e4b7c868858329928c7fb25f4cac881d81458a99/packages/ERTP/core/unitOps.js#L4
+
https://github.com/Agoric/agoric-sdk/blob/e4b7c868858329928c7fb25f4cac881d81458a99/packages/ERTP/core/unitOps.js#L24-L28

But listExtentOps cannot really be exposed as is, since it requires to provide 3 methods to be instanciated

https://github.com/Agoric/agoric-sdk/blob/e4b7c868858329928c7fb25f4cac881d81458a99/packages/ERTP/core/config/extentOps/listExtentOps.js#L9-L11

I'm making an issue on this topic

Reply:
Issue: https://github.com/Agoric/agoric-sdk/issues/330

while writing the issue, i realized that it's not possible to use ListExtentOps without changing first how the mint API works

I'm keeping `uniExtentOps` for now, pending a discussion on ListExtentOps

Reply:
@katelynsills github hides this discussion for me, so i'm not sure you've seen it

Reply:
Ok thanks, I saw it. Sticking with uniExtentOps makes sense to me for those reasons. I'm ok publishing this before we fix ListExtentOps, but that should be a next step